code/modules/mapping/mapping_helpers.dm

```diff
@@ -850,23 +857,25 @@ INITIALIZE_IMMEDIATE(/obj/effect/mapping_helpers/no_lava)
 	admin_spawned = TRUE
 
 /obj/effect/mapping_helpers/dead_body_placer/LateInitialize()
-	var/area/a = get_area(src)
-	var/list/trays = list()
-	for (var/i in a.contents)
-		if (istype(i, /obj/structure/bodycontainer/morgue))
-			if(admin_spawned)
-				var/obj/structure/bodycontainer/morgue/early_morgue_tray = i
-				if(early_morgue_tray.connected.loc != early_morgue_tray)
-					continue
-			trays += i
-	if(!trays.len)
+	var/area/morgue_area = get_area(src)
+	var/list/obj/structure/bodycontainer/morgue/trays = list()
+	for(var/turf/area_turf as anything in morgue_area.get_contained_turfs())
+		var/obj/structure/bodycontainer/morgue/morgue_tray = locate() in area_turf
```

Comment:
do u want to make it not spawn in the secure morgue trays for me 🛩️

Reply:
sure